sdk/program/src/vote/instruction.rs

```diff
@@ -251,7 +251,7 @@ pub struct CreateVoteAccountConfig<'a> {
 impl<'a> Default for CreateVoteAccountConfig<'a> {
     fn default() -> Self {
         Self {
-            space: VoteStateVersions::vote_state_size_of(false) as u64,
```

Comment:
Is this only used in tests? Otherwise seems like we will create different sized vote states

Reply:
probably should not have had this default. It was being explicitly overriden with the value of the feature flag for use in the cli and tests. However there is 2 instruction creators (deprecated) that use the default:
https://github.com/anza-xyz/agave/blob/5eab6ae2b5aec6910bd5a00bfccc00614450bec7/sdk/program/src/vote/instruction.rs#L260-L301

In theory a downstream project could be using these creators for their program and this change has the potential to break them.

Reply:
We have to either leave this `default()` implementation as is, and ensure that our codebase overrides it, or first remove the 2 deprecated instruction creaters and then update the `default()` to the new size.